src/shims/panic.rs

```diff
@@ -39,7 +39,6 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         trace!("miri_start_panic: {:?}", this.frame().span);
 
         if this.tcx.tcx.sess.panic_strategy() == PanicStrategy::Abort  {
-            // FIXME: Actually print out the payload here
             throw_unsup_format!("the evaluated program panicked");
```

Comment:
Some FIXME still seems appropriate as "unsup" is wrong... this is basically an abort, right? So you can copy the intrinsic from there.